examples/Client/StateManagement/Program.cs

```diff
@@ -24,6 +24,7 @@ class Program
             new StateStoreExample(),
             new StateStoreTransactionsExample(),
             new StateStoreETagsExample(),
+            new BulkStateExample()
```

Comment:
Make sure you add a reference to this existing at the bottom of the README.

https://github.com/dapr/dotnet-sdk/blob/master/examples/Client/StateManagement/README.md